docs/howto/distributed-tracing/proc_tracing-single-service.adoc

```diff
@@ -0,0 +1,85 @@
+
+[id='tracing-single-service_{context}']
+= Tracing a single service
+
+In the first example, we'll show how to trace a standalone "hello world" service.
+This service doesn't make any external invocations, so the results won't be particularly interesting, but we will build on this example later.
+
+.Prerequisites
+
+* The Jaeger tracer running on `localhost`.
++
+--
+[source,bash]
+----
+docker run -it --rm -p 6831:6831/udp -p 16686:16686 jaegertracing/all-in-one
+----
+
+Wait until the `Connected to peer` message shows in the console.
+
+--
+
+.Procedure
+
+. Include the `microprofile-opentracing` and `jaeger` fractions in your `pom.xml`.
++
+--
+[source,xml]
+----
+<dependencies>
+include::pom.xml[tag=mp-opentracing,indent=2]
+include::pom.xml[tag=jaeger,indent=2]
+</dependencies>
+----
+--
+
+. Configure the Jaeger tracer in `project-defaults.yml`.
++
+--
+[source,yaml]
+----
+swarm:
+include::src/main/resources/project-defaults.yml[tag=jaeger,indent=2]
```

Comment:
IntelliJ IDEA renders all includes just fine except of this one. @tradej Do you maybe see an issue here?